drivers/dma/dma_stm32.c

```diff
@@ -84,19 +103,26 @@ static void dma_stm32_irq_handler(void *arg)
 	if (func_ll_is_active_tc[id](dma)) {
 		func_ll_clear_tc[id](dma);
 		stream->dma_callback(stream->callback_arg, id + STREAM_OFFSET,
-					0);
+			0);
 	} else if (stm32_dma_is_unexpected_irq_happened(dma, id)) {
 		LOG_ERR("Unexpected irq happened.");
 		stream->dma_callback(stream->callback_arg, id + STREAM_OFFSET,
-					-EIO);
+			-EIO);
 	} else {
 		LOG_ERR("Transfer Error.");
 		dma_stm32_dump_stream_irq(dev, id);
 		dma_stm32_clear_stream_irq(dev, id);
 
 		stream->dma_callback(stream->callback_arg, id + STREAM_OFFSET,
-					-EIO);
```

Comment:
To be moved to commit: drivers: dma: configuration of the dma_stm32
But yet the alignment is not correct, this could come from you editor setting (1 tab equals 8 spaces)

Reply:
moved to a07f54a2f2d76d1c286b46ed172a7d587c534e16